src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/yml/mailer.yml

```diff
@@ -6,3 +6,7 @@ framework:
             recipients:
                 - redirected@example.org
                 - redirected1@example.org
+        headers:
+            from: from@example.org
```

Comment:
Will it be possible to use `from: Firstname Lastname <from@example.org>`?

Reply:
Yes, it does work.

Reply:
Or an array if you want multiple From.

Reply:
That's awesome, nice!